static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-row-support.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { makeABTest } from 'common/modules/commercial/contributions-utilities';
+
+import {
+    getSupporterPaymentRegion as geolocationGetSupporterPaymentRegion,
+    getSync as geolocationGetSync,
+} from 'lib/geolocation';
+
+import config from 'lib/config';
+
+import type { CtaUrls } from 'common/modules/commercial/contributions-utilities';
+
+const ROWsupportURL = 'https://support.theguardian.com/int';
+const abTestName = 'AcquisitionsEpicRowSupport';
+
+const oneButtonTemplate = (urls: CtaUrls): string => {
+    const url = urls.supportUrl || '';
+
+    const supportButtonSupport = `
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-single-button"
+              href="${url}"
+              target="_blank">
+              Support The Guardian
+            </a>
+        </div>`;
+
+    const paymentLogos = `<img class="contributions__payment-logos contributions__contribute--epic-member" src="${config.get(
```

Comment:
We have paymentLogos and supportButtonSupport in the AU test we are currently running. Rather than have this template code in 2 places can we pull it out? It will keep the test files nice and tidy